### PR TITLE
chore: Compose UI Minor Fixes (1.5 cherrypicks to main)

### PR DESCRIPTION
**Pull request by mairin**
### What does this PR do?

There's some compose UI minor tweaks I made in 1.5 that, due to the point of the release process I made them and not being git proficient 😅 I pushed to 1.5.x branch and NOT main. So here I am cherry picking them back into main. :)

### CODE REVIEW

File: extensions/compose/package.json [hunk 1]
```diff
@@ -50,12 +50,12 @@
           "content": [
             [
               {
-                "value": "Compose support in Podman Desktop enables you to use Compose specification CLI implementations such as `podman compose` and `docker compose` with supported container engines."
+                "value": "Podman Desktop provides optional command line support for compose files with Podman."
```
Comment (jeffmaury) [inline extensions/compose/package.json]: The Compose spec always references Compose with a capital C

File: extensions/compose/src/extension.ts [hunk 2]
```diff
@@ -192,8 +192,6 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     },
   };
 
-  providerOptions.emptyConnectionMarkdownDescription = composeDescription;
-
```
Comment (benoitf) [inline extensions/compose/src/extension.ts]: AFAIK it should not be removed ?
it's in 1.5.x branch as well
https://github.com/containers/podman-desktop/blob/1.5.x/extensions/compose/src/extension.ts#L188

Comment (mairin) [reply extensions/compose/src/extension.ts]: 🤦🏻‍♀️ omg great catch, was an overzealous cut when i was fixing the merge conflict, fixing

File: extensions/compose/package.json (hunk 1 above)
Comment (jeffmaury) [inline extensions/compose/package.json]:
```suggestion
                "value": "Podman Desktop provides optional command line support for Compose files with Podman."
```

**Review by benoitf — APPROVED** [review]

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**